src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -175,6 +204,50 @@ const EditShiftHours = ({
     );
   };
 
+  const hasShiftMovedFromTimecard = (startDate, endDate) => {
+    const startDayTime = dayjs(timecardDate).startOf('day');
+    const endDayTime = dayjs(timecardDate).endOf('day');
+    const currentDayStart = new Date(startDayTime).getTime();
+    const currentDayEnd = new Date(endDayTime).getTime();
+    const startDay = new Date(startDate).getTime();
+    const endDay = new Date(endDate).getTime();
+
+    const singleDateMoved =
+      !finishEntryExists &&
+      (startDay < currentDayStart || startDay > currentDayEnd);
+
+    const bothDatesMoved =
+      finishEntryExists &&
+      (startDay > currentDayEnd || endDay < currentDayStart);
```

Comment:
I'm not 100% sure on the second part of the logic for these variables, might be one to have a quick call on?

Reply:
I've tried various scenarios but can't make this fail.

Reply:
I set up a shift with the start and end on 17/03/23. I then moved the start date to be 16/03/23 and no notification was displayed, which is correct.

Reply:
Resolving as we think this is covered by EAHW-2279 - will double check functionality with BAs then